css/properties.json

```diff
@@ -4667,6 +4667,21 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "justify-items": {
+    "syntax": "normal | stretch | <baseline-position> | [ <overflow-position>? <self-position> ] | [ legacy || [ left | right | center ] ]",
```

Comment:
It looks as if this has changed: https://drafts.csswg.org/css-align-3/#propdef-justify-items. If you agree, and can update it, I'll merge this PR.